Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
     racc (1.6.0)
-    rack (2.2.3)
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
-    rails (7.0.2.3)
-      actioncable (= 7.0.2.3)
-      actionmailbox (= 7.0.2.3)
-      actionmailer (= 7.0.2.3)
-      actionpack (= 7.0.2.3)
-      actiontext (= 7.0.2.3)
-      actionview (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activestorage (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    rack (2.2.4)
+    rack-test (2.0.2)
```

Comment:
### [rack-test](https://github.com/rack/rack-test)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.1.0 → 2.0.2

[_[change-log](https://github.com/rack/rack-test/blob/main/History.md), [source-code](https://github.com/rack/rack-test), [gem-diff](https://my.diffend.io/gems/rack-test/1.1.0/2.0.2)_]





<details>
<summary>Commits</summary>

A change of **100** commits. See the full changes on [the compare page](https://github.com/rack/rack-test/compare/v1.1.0...v2.0.2).

These are the first 10 commits:
- (7a94af8) [Sets up focussed tests and fixes inconsistency (#237)](https://github.com/rack/rack-test/commit/7a94af8887f8cd523bea55092461bad29040abbf)
- (6c07bf5) [Fix: Fixes digest authentication with Cookies (#235)](https://github.com/rack/rack-test/commit/6c07bf53adf4a4082e206561f5b9b43142fdc821)
- (aee2990) [Add Ruby 2.6 (#240)](https://github.com/rack/rack-test/commit/aee2990c114582efc564979050b7f01e909de391)
- (f2b9179) [Update Bundler to the latest version if Ruby is >= 2.3 (#242…](https://github.com/rack/rack-test/commit/f2b9179062c18b270757e9b6522996c7c1c9b480)
- (dcdd908) [Pin the dependency sinatra version temporarily. (#244)](https://github.com/rack/rack-test/commit/dcdd908ff825a54a7f3f8a30c86547fa997693e2)
- (02d5185) [Fix: Respect Content-Type of multipart requests (#238)](https://github.com/rack/rack-test/commit/02d51851da38eb23207a61b587ddcfe7bc81ecea)
- (5f4f970) [Add IRC notification for a Travis CI build failure. (#246)](https://github.com/rack/rack-test/commit/5f4f970bca32ea15b4423c5bc8713d7b36d003ad)
- (a2f762d) [Retry to download code climate. (#245)](https://github.com/rack/rack-test/commit/a2f762d5abc2ead277dab51794d22083b72809ea)
- (37eba28) [Fix broken jruby 9.1 tests with openjdk8 (#251)](https://github.com/rack/rack-test/commit/37eba28157eda33b7ca7550cc6d9c30fbae660db)
- (8a3ef19) [Improve stale bot setting [ci skip] (#254)](https://github.com/rack/rack-test/commit/8a3ef198cdb747428e4ba36cc818f2bc79ebac30)


</details>